ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>"Google cloud" -&gt; "Google Cloud" (for branding)</p><p> </p><p>fixed typo and clarified that it's a dataset</p><p>"...the movie recommentation as an example..." -&gt; "...a move recommendation dataset as an example..."</p><p> </p><p>"into the featurestore service." -&gt; "into Feature Store." OR "into a featurestore." (if you want to refer to the featurestore resource)</p><p> </p><p>"How to serve online prediction requests..." (make "request" plural)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>